resources/assets/uad_lists.json

```
@@ -3737,7 +3737,7 @@
   {
     "id": "com.samsung.android.dynamiclock",
     "list": "Oem",
-    "description": "Samsung Dynamic Lock\nAutomatically changes your Lock screen’s wallpaper\nhttps://www.samsung.com/us/support/answer/ANS00084210/\n",
+    "description": "Samsung Wallpaper services (AKA \"Dynamic Lock\")\nAutomatically changes your Lock screen’s wallpaper\nhttps://www.samsung.com/us/support/answer/ANS00084210/\n",
```

```suggestion
    "description": "Samsung Wallpaper services (AKA 'Dynamic Lock')\nAutomatically changes your Lock screen’s wallpaper\nhttps://www.samsung.com/us/support/answer/ANS00084210/\n",
```

In the future, you don't need to escape it anymore. You can just use single quotes; that works too, as long as you don't use double ones to end the description field.